src/plugin/transform.ts

```diff
@@ -136,23 +158,24 @@ function insertTabs(
         tabClose.block = true;
         tabPanelOpen.block = true;
         tabPanelClose.block = true;
-        tabOpen.attrSet('id', tabId);
-        tabOpen.attrSet('class', 'yfm-tab');
+        tabOpen.attrSet(TAB_DATA_ID, tabId);
```

Comment:
I decided not use the `id` attribute to avoid `navigation` affecting (if a collision exists on the page).